model/src/pyrenew/latent/i0.py

```diff
@@ -0,0 +1,77 @@
+import numpyro as npro
+import numpyro.distributions as dist
+from pyrenew.metaclass import RandomVariable
+
+
+class Infections0(RandomVariable):
+    """Initial infections helper class.
+
+    It creates a random variable for the initial infections with a prior
+    distribution.
+    """
+
+    def __init__(
+        self,
+        name: str = "I0",
+        I0_dist: dist.Distribution = dist.LogNormal(0, 1),
```

Comment:
I do not think this should have a default prior.

Reply:
Addressed in 33d9ed0